app/scripts/routes.js

```diff
@@ -16,6 +16,10 @@ define([], function() {
                 templateUrl: 'views/students/index.html',
                 controller: 'StudentsIndexCtrl',
                 relativePath: '/students'
+            },
+            '/students/:docket/update': {
```

Comment:
I first marked it as _conflict_, since both StudentsUpdate and StudentsEdit were added to the documentation, but only StudentsUpdate exists in API.
I now deleted the resources marked as _conflict_, to clear the documentation.
Finally, I changed the view's path to /update to maintain consistency. This last change is of course, not required since API and Front Paths don't need to match.
Why would you keep /edit?